lib/adb.js

```diff
@@ -494,16 +494,21 @@ ADB.prototype.checkApkCert = function (apk, pkg, cb) {
 };
 
 ADB.prototype.checkCustomApkCert = function (apk, pkg, cb) {
+  var self = this;
```

Comment:
instead of aliasing `this` to `self`, we simply `.bind(this)` any function which needs the `this` context